bin/README.md

```diff
@@ -0,0 +1,18 @@
+## CLI for BatChain P2P File Storage
```

Comment:
What do you think about including the contents of this new readme in the existing one? The command line is different from the library, but I think it might be simpler to have all documentation in one place.

Reply:
Yep, we can definitely do that!